Readme.md

```diff
@@ -73,6 +73,8 @@ Welcome all of you future star candidates who want to strive to attend the acade
   > As an academician.
 - [Mohammad Fiqri Firmansyah](https://github.com/TakanashaTaryu) [RYU]
   > As an academician.
+-[Umar Zaki Gunawan](https://github.com/marzkigun27). [UZY]
```

Comment:
Wrong format, look again carefully, and please fix it, thanks

Reply:
forgive my lack of attention, I've already corrected it, please check.

Reply:
Sure, no problem 👌